pkg/apis/datadoghq/v1alpha1/datadogagent_types.go

```diff
@@ -100,6 +100,36 @@ type DatadogAgentSpecAgentSpec struct {
 	// Update strategy configuration for the DaemonSet
 	DeploymentStrategy *DaemonSetDeploymentStrategy `json:"deploymentStrategy,omitempty"`
 
+	// AdditionalAnnotations provide annotations that will be added to the Agent Pods.
+	AdditionalAnnotations map[string]string `json:"additionalAnnotations,omitempty"`
+
+	// AdditionalLabels provide labels that will be added to the cluster checks runner Pods.
+	AdditionalLabels map[string]string `json:"additionalLabels,omitempty"`
+
+	// If specified, indicates the pod's priority. "system-node-critical" and "system-cluster-critical"
+	// are two special keywords which indicate the highest priorities with the former being the highest priority.
+	// Any other name must be defined by creating a PriorityClass object with that name. If not specified,
+	// the pod priority will be default or zero if there is no default.
+	PriorityClassName string `json:"priorityClassName,omitempty"`
+
+	// Specifies the DNS parameters of a pod.
+	// Parameters specified here will be merged to the generated DNS
+	// configuration based on DNSPolicy.
+	// +optional
+	DNSConfig *corev1.PodDNSConfig `json:"dnsConfig,omitempty"`
+
```

Comment:
could you add a dns policy field as well? https://godoc.org/k8s.io/api/core/v1#DNSPolicy

Reply:
done